interfaces/Referral-App/src/app/referral/referral.page.html

```diff
@@ -81,11 +82,31 @@
   </div>
 </ion-content>
 
-<ion-footer>
+<ion-content *ngIf="!isSupportedRegion()">
+  <ion-card *ngIf="!category">
+    <ion-card-header>
+      <ion-card-title>
+        The Netherlands Red Cross provides helpful information for
+        <span
+          *ngFor="let supportedRegion of supportedRegions; first as isFirst; last as isLast"
+          >{{ isFirst ? '' : (isLast ? ' and' : ',')}}
+          <a routerLink="/{{toSnakeCase(supportedRegion)}}"
```

Comment:
The right syntax would be:
```suggestion
          <a [routerLink]="['/', toSnakeCase(supportedRegion)]"
```